core/Names.cc

```diff
@@ -95,6 +98,8 @@ string Name::toString(const GlobalState &gs) const {
                 return fmt::format("<Class:{}>", this->unique.original.data(gs)->show(gs));
             } else if (this->unique.uniqueNameKind == UniqueNameKind::Overload) {
                 return absl::StrCat(this->unique.original.data(gs)->show(gs), " (overload.", this->unique.num, ")");
+            } else if (this->unique.uniqueNameKind == UniqueNameKind::DefaultArg) {
+                return fmt::format("{}<defaultAarg>{}", this->unique.original.data(gs)->show(gs), this->unique.num);
```

Comment:
I just noticed a typo in the format string here: `defaultAarg` should be `defaultArg` I believe.